pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -69,11 +221,54 @@ func GetBootstrapSecurityContextConstraints(buildControllerUsername string) []ka
 				// by the admission controller.  Admission will first look for the SupplementalGroupsAnnotation
 				// on the namespace and if it is unable to find that annotation it will attempt
 				// to use the UIDRangeAnnotation.  If neither annotation exists then creation
-				// of the SCC will fail.
+				// of the strategy will fail.
 				Type: kapi.SupplementalGroupsStrategyMustRunAs,
 			},
-			Groups: []string{AuthenticatedGroup},
 		},
+		// SecurityContextConstraintsAnyUID allows no host access and allocates SELinux.
```

Comment:
I think this is the summary:
* `system:authenticated` should remain in the `restricted` SCC for now
* `system:cluster-admins` should be in `privileged` and `anyuid`
* `anyuid` should have `priority` of `1`, others should have `priority` of `0`. This means anyone with access to the `anyuid` SCC will not have uids assigned to their pods automatically.

@smarterclayton does that sound right? note that the `anyuid` SCC would only apply to pods the cluster admin created directly, not ones created via rc or dc. If they wanted those to use `anyuid`, they'd have to add the service account(s) they wanted to run as any uid to the SCC